.github/workflows/CI_CD.yml

```diff
@@ -0,0 +1,26 @@
+name: CI/CD 
+on:
+  push:
+    
+jobs:
+  model-eval-promotion:
+    runs-on: ubuntu-latest
+    env:
+      NEPTUNE_API_TOKEN: 'ANONYMOUS' #${{ secrets.NEPTUNE_API_TOKEN }} 
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r CI_CD/requirements.txt
+      - name: Model Promotion(CI/CD)
+        run: python CI_CD/scripts/model_promotion.py
+      - name: Deployment Status
+        if: success()
```

Comment:
this looks artificial. 

perhaps we could simply drop this step and just have one step "Model Promotion (CI/CD)" ?